flask_rest_jsonapi/decorators.py

```diff
@@ -79,7 +79,7 @@ def wrapper(*args, **kwargs):
                                  e.status,
                                  headers)
         except Exception as e:
-            if current_app.config['DEBUG'] is True:
+            if current_app.config['DEBUG'] is True or current_app.config['PROPAGATE_EXCEPTIONS'] is True:
```

Comment:
This will fail for every app which has not set `PROPAGATE_EXCEPTIONS`
```suggestion
            if current_app.config['DEBUG'] is True or current_app.config.get('PROPAGATE_EXCEPTIONS') is True:
```